templates/frontend/objects/article_details.tpl

```diff
@@ -141,25 +141,25 @@
 
 			{* Keywords *}
 			{if !empty($publication->getLocalizedData('keywords'))}
-			<div class="item keywords">
-				<span class="label">
+			<section class="item keywords">
+				<h2 class="label">
 					{capture assign=translatedKeywords}{translate key="article.subject"}{/capture}
 					{translate key="semicolon" label=$translatedKeywords}
-				</span>
+				</h2>
 				<span class="value">
 					{foreach name="keywords" from=$publication->getLocalizedData('keywords') item="keyword"}
 						{$keyword|escape}{if !$smarty.foreach.keywords.last}{translate key="common.commaListSeparator"}{/if}
 					{/foreach}
 				</span>
-			</div>
+			</section>
```

Comment:
We should add `<section>` and `<h2>` to the DOI section that's just above keywords as well.

Once you've done that, you'll need to add some CSS code to bring the size of the DOI/Keywords labels back down to `14px` and make them inline, so that they look like this:

![Selection_173](https://user-images.githubusercontent.com/2306629/70634503-3ca6f400-1c2a-11ea-94b7-5fdda239dd89.png)

To do that, you'll need to add some CSS that just modify's the `.label` styles for the `doi` and `keywords` sections. Find this block in `article_details.less`:

```css
	.main_entry {

		.item {

			.label {
				margin: 0 0 @double;
				font-family: @font-heading;
				font-size: @font-bump;
				font-weight: @bold;
			}
		}
```

You can extend the `.item` definition like this:

```css
.item {
	
	.label {
		margin: 0 0 @double;
		font-family: @font-heading;
		font-size: @font-bump;
		font-weight: @bold;
	}

	&.doi .label,
	&.keywords .label {
		display: inline;
		font-size: @font-base;
	}
}
```

That will bring just those labels down to size.